rmgpy/molecule/molecule.py

```diff
@@ -1595,4 +1595,38 @@ def toGroup(self):
         
         return group
 
-
+    def getNthNeighbor(self, startingAtoms, n, blacklist):
```

Comment:
Try changing the variable name from 'blacklist' to 'ignore', as that is more clear. My bad for the stupid terminology.  I suggest making 'ignore' an empty list by default:

`def getNthNeighbor(self, startingAtoms, n, ignore = []):`

Then, be a little bit more specific on defining each variable: E.g. startingAtoms is a list of :class:Atom for which we will get the nth partner, n is an interger, ignore is a list of :class:Atoms that will not be returned in the neighboring atoms. etc.